### PR TITLE
Add technology_types_explanors folder with spatial_technologies.md file

### PR DESCRIPTION
**Pull request by ami-day**
Added a new folder with simple and clear explanatory documents (.md) files about the common technology types that we curate (scATAC-seq, CITE-seq, 10X Visium) and the technology-specific curation requirements, if any.

### CODE REVIEW

File: docs/technology_types_guide/CITE_seq/Overview.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Specifically, a set of DNA-barcoded antibodies are bound to specific cell surface protein(s). The cell and bound antibody are then encapsulated in a droplet alongside a poly-dT oligo-coated bead, and the cell is lysed. Reverse transcription and cDNA amplification of the antibody-bound DNA barcode and cellular mRNA transcripts then occurs, and transcripts are indexed with a cell barcode. This is illustrated below in **Figure 1**.
+
+Once scRNA-seq libraries are sequenced, transcripts can be mapped back to individual cells using the cell barcode. The presence of a particular single cell surface protein can also be identified by the antibody barcode. The scRNA-seq library preparation method used for CITE-seq is typically 10X Genomics, Drop-seq or other oligodT-based scRNA-seq methods.
+
+
+<center><u>Figure 1</u>: Illustration of CITE-seq</center>
+<br
+    </br>
+
+<img src="https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/Add-technology-type-info-folder/technology_types_explanors/CITE_seq/visuals/CITE-seq.png" width="700" height="500">
```
Comment (amnonkhen) [inline docs/technology_types_guide/CITE_seq/Overview.md:16]: 1. This link is broken.
2. Please use relative links
3. Verify all other image links.

File: docs/technology_types_guide/CITE_seq/Overview.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+- An HTO antibody index index file which has the format “feature_barcode, feature_name". This should include the DNA barcode sequence and the antibody name, antibody ID or multiplex HTO sample ID. The mapped barcode counts might also be included in this file. See example dataset [here](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/Add-technology-type-info-folder/technology_types_explanors/CITE_seq/example_datasets/GSM2895283_Hashtag-HTO-count.csv).
+
+- Raw ADT sequencing data with an ADT barcode sequence. ADT data are mapped to a typically 10-12 bp reference ADT list separately.
+
+- An ADT antibody index index file which has the format “feature_barcode, feature_name". This should include the DNA barcode sequence and the antibody name, antibody ID or multiplex HTO sample ID. The mapped barcode counts might also be included in this file. See example dataset [here](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/Add-technology-type-info-folder/technology_types_explanors/CITE_seq/example_datasets/GSM2895283_Hashtag-ADT-1-count.csv).
```
Comment (amnonkhen) [inline docs/technology_types_guide/CITE_seq/Overview.md:57]: relative link for file

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: docs/technology_types_guide/CITE_seq/Overview.md (hunk 1 above)
Comment (ESapenaVentura) [reply docs/technology_types_guide/CITE_seq/Overview.md:16]: @amnonkhen relative links behave poorly in github.io (https://ebi-ait.github.io/hca-ebi-wrangler-central/ ,where this SOP will be shown)

File: docs/technology_types_guide/10X_Visium/Curation_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- Image file(s) associated with each imaged specimen should be included as a supplementary file in the Imaged specimen tab.
+
+- At least 1 image file per imaged specimen should be overlaid with 10X Visium image coordinates. [See detected_tissue_image.jpg.](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/Add-technology-type-info-folder/technology_types_explanors/10X_Visium/example_dataset/detected_tissue_image.jpg)
+
+### (4) Analysis file(s)
+
+- For each imaged specimen, there should be a list of 10X Visium spatial barcodes linked to the image coordinates. [See tissue_positions_list.csv.](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/Add-technology-type-info-folder/technology_types_explanors/10X_Visium/example_dataset/tissue_positions_list.csv)
+
+- If the imaged specimen(s) have been annotated by a contributor (e.g. organ-specific anatomical parts), ideally, we should include a file linking   the annotations to the image coordinates.
```
Comment (ESapenaVentura) [inline docs/technology_types_guide/10X_Visium/Curation_guidelines.md:40]:
```suggestion
- If the imaged specimen(s) have been annotated by a contributor (e.g. organ-specific anatomical parts), ideally, we should include a file linking the annotations to the image coordinates.
```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
I have left some comments:

Overall the instructions seem good to me, I have nothing to say about the contents.

However, I find the organization of the files and instructions a bit confusing. Overall:

- Why generate multiple SOPs, one per each technology? It would be easier to find if we had everything condensed in a single SOP with a table of contents (Auto-generated in github.io)
- Why have it outside of our SOPs?

I think this would look better re-structured into a single SOP, and also please use the assets folder for the supporting information!

Also another general comment: Please don't use branches as sources for references, as branches can easily be deleted. If you'd like to reference an image/file, please use the master branch to generate the hyperlink (Even if it's not yet available due to it not being merged)

Finally, if you want to show images, you can use an image hyperlink: e.g.
detected_tissue_image

Notice that I changed "blob" to "raw" for github to be able to load the actual image

Let me know if you have any question about my review!

File: docs/technology_types_guide/10X_Visium/Curation_guidelines.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Required metadata are labelled in the template spreadsheet. Some metadata is required only if the tab is present in the spreadsheet
+and not left empty. For 10X Visium data, the following tabs should be filled:
+
+- Imaged specimen
+- Imaging preparation protocol
+- Imaging protocol
+
+### (2) Ontology terms
+
+- See [technology type table](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/documentation_add_tech_type_table/technology_type_table.md)
```
[the remark this answers is not in the document]
Comment (ami-day) [reply docs/technology_types_guide/10X_Visium/Curation_guidelines.md:28]: yes, it is meant to be for all technology types so should stay separate.

**Review by ami-day — COMMENTED** [review]
Made suggested changes - added absolute link to master branch.

**Pull request merged**